textile/digital_printing/report/fabric_printing_summary/fabric_printing_summary.json

```diff
@@ -0,0 +1,32 @@
+{
+ "add_total_row": 1,
+ "columns": [],
+ "creation": "2023-08-15 16:22:43.702795",
+ "disable_prepared_report": 0,
+ "disabled": 0,
+ "docstatus": 0,
+ "doctype": "Report",
+ "filters": [],
+ "idx": 0,
+ "is_standard": "Yes",
+ "modified": "2023-08-15 19:27:51.714706",
+ "modified_by": "Administrator",
+ "module": "Digital Printing",
+ "name": "Fabric Printing Summary",
+ "owner": "Administrator",
+ "prepared_report": 0,
+ "ref_doctype": "Print Order",
+ "report_name": "Fabric Printing Summary",
+ "report_type": "Script Report",
+ "roles": [
+  {
+   "role": "Manufacturing User"
+  },
+  {
+   "role": "Stock User"
+  },
+  {
+   "role": "Accounts User"
+  }
+ ]
```

Comment:
Need to decide permissions. Lets give permission to Sales Manager, Accounts User and Manufacturing Manager only?